code/modules/research/rdconsole_tgui.dm

```diff
@@ -423,6 +423,14 @@
 						to_chat(usr, "<span class='notice'>The destructive analyzer appears to be empty.</span>")
 						return
 
+					if(istype(linked_destroy.loaded_item,/obj/item/stack))//Only deconsturcts one sheet at a time instead of the entire stack
+						var/obj/item/stack/ST = linked_destroy.loaded_item
+						if(ST.get_amount() < 1)
+							playsound(linked_destroy, 'sound/machines/destructive_analyzer.ogg', 50, 1)
+							qdel(ST)
+							linked_destroy.icon_state = "d_analyzer"
+							return TRUE
+
```

Comment:
Why can the DA take sheets at all?

Reply:
Because many sheets carry science values. Mostly related to 'materials' research, but also some others, like medical stacks or extra advanced/cool materials.

Reply:
I see. Well, your code here doesn't appear to ever reduce the amount in the stack.

Reply:
Nope, the code for that exists already and happens later in same proc. This is basically a contingency check for "do we even have enough of a stack for it to contribute?"